5 Arrays and Cycles.js

```diff
@@ -1,99 +1,90 @@
-/*Дан массив имен ваших знакомых, добавить к каждому элементу массива слова hello*/
+//Дан массив имен ваших знакомых, добавить к каждому элементу массива слова hello
 
-let arr = ["Olga", "Roman", "Yana", "Oleg",];
-let result = arr.map(function(item, index, array) {
-    return ('hello ' + item);
+let arr = ["Olga", "Roman", "Yana", "Oleg"];
+let result = arr.map(function (item, index, array) {
+  return "hello " + item;
 });
 console.log(result);
 
+/*Дан массив состоящий из названий фильмов, выполните перебор массива 
+с выводом в консоль названия каждого фильма*/
 
-/*Дан массив состоящий из названий фильмов, выполните перебор массива с выводом в консоль названия каждого фильма*/
-
-let arr = ['matrix', 'die hard', 'snatch', 'shrek 3'];
-for (let i=0; i<arr.length; i++){
-    console.log(arr[i]);
-};
-
+let arr = ["matrix", "die hard", "snatch", "shrek 3"];
+for (let i = 0; i < arr.length; i++) {
+  console.log(arr[i]);
+}
 
 //Дан массив производителей автомобилей, преобразовать массив в строку и обратно в массив
 
 let arr = ["audi", "bmw", "fiat", "lada"];
-let str = arr.join(',');
+let str = arr.join(",");
 console.log(str);
-arr = str.split(',');
+arr = str.split(",");
 console.log(arr);
 
-
 // Преобразовать числовой массив в Boolean
-let arr = [1,0,7,8,0,0,5,6];
-let result = arr.map(function(item, index, array){
-    return Boolean(item);
+let arr = [1, 0, 7, 8, 0, 0, 5, 6];
+let result = arr.map(function (item, index, array) {
+  return Boolean(item);
 });
 console.log(result);
 
 //Отфильтровать массив [1,6,7,8,3,4,5,6] по значению >3
 
-let arr = [1,6,7,8,3,4,5,6];
-let result = []
-for (let i=0; i<arr.length; i++){
-    if (arr[i]>3){
-        result.push(arr[i]);
-    };
-};
+let arr = [1, 6, 7, 8, 3, 4, 5, 6];
+let result = [];
+for (let i = 0; i < arr.length; i++) {
+  if (arr[i] > 3) {
+    result.push(arr[i]);
+  }
+}
```

Comment:
Do you know about method `filter()`? https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/filter